README.md

```diff
@@ -1,34 +1,38 @@
-# mark-fullstack-proj--starter-1
-
-## Usage:
-
-Instead of cloning this project, click "Use this template". This will allow you to create a repo on github which has this project's content but which is not a fork of it.
-
-Make sure you create the repo as being owned by your own account not by the WeAreAcademy organisation.
+# C7C1 Study Catalog - Backend
 
 ## Install
 
 `yarn`
 
 ## DB Setup
 
-Copy .env.example to .env and set `DATABASE_URL`, `LOCAL_DATABASE_URL` and `PORT` to your liking.
+-   Copy .env.example to .env and set `DATABASE_URL`, `LOCAL_DATABASE_URL` and `PORT` to your liking.
+-   You will need to create a user in the Rapid API website to obtain the `X_RAPIDAPI_KEY` and be able to get images for the thumbnails (if not, you will still get a standard image when the user creates a new recommendation). Please refer to https://docs.rapidapi.com/.
+-   Please link the corresponding `DISCORD_URL` in order to receive a notification when a new recommendation is posted.
```

Comment:
- Please link the corresponding discord webhook to `DISCORD_URL` in order to receive a notification when a new recommendation is posted.